braze/client.py

```diff
@@ -31,15 +44,13 @@ class BrazeClient(object):
     DEFAULT_API_URL = 'https://rest.iad-02.braze.com'
     USER_TRACK_ENDPOINT = '/users/track'
     USER_DELETE_ENDPOINT = '/users/delete'
-
-    REQUEST_POST = 'post'
+    MAX_RETRIES = 3
+    MAX_WAIT_SECONDS = 5
 
     def __init__(self, api_key, api_url=None):
         self.api_key = api_key
         self.api_url = api_url or self.DEFAULT_API_URL
-        self.requests = requests
```

Comment:
This was set by previous author to make it easier to monkeypatch requests, however now we monkeypatch requests with `requets_mock`.